submissions/adeyemiJuwon/dom/newsletter.js

```diff
@@ -1,35 +1,22 @@
 let submitForm = document.getElementById('newsletter');
 let email = document.getElementById('email');
+let emailRegex = /^\w+[\w-\.]*\@\w+((-\w+)|(\w*))\.[a-z]{2,3}$/;
 
 
-
-function required(field, event){
-    if(field.value == ""){
-        document.querySelector(".error").innerHTML = "Please enter your Email Address";
-        event.preventDefault();
-        return false;
-    }else{
-        document.querySelector(".error").innerHTML = "";
-        return true;
-    }
-    }
-    
-    function emailFieldValidation(event){
-        emailRegex = /^\w+[\w-\.]*\@\w+((-\w+)|(\w*))\.[a-z]{2,3}$/;
-        if (!email.value.match(emailRegex)){
+    function validateForm(e){
+        if(!email.value.match(emailRegex)){
             document.querySelector(".error").innerHTML = "Please enter a valid email";
             event.preventDefault();
```

Comment:
This should be `e` not `event`. Except you want to pass `event` as the parameter

Reply:
Yes I thought both can work together. either e or event should work the same way.